### DISCUSSION TITLE
Disabling SAO/ALF in vvenc

### DISCUSSION THREAD

**Discussion by birdie-github** [fraunhoferhhi/vvenc#388] [Q&A]
Is it possible just to disable SAO or it's now part of ALF?

Anyways how to disable SAO or if it's not possible, to disable both?

**Comment by birdie-github** [comment]
Explained [here](https://github.com/fraunhoferhhi/vvenc/discussions/203#discussioncomment-3837464).

**Reply by adamjw24** [reply]
SAO: `--SAO=0`
ALF: `--ALF=0 --CCALF=0`
Prefiltering: `--MCTF=0`
